<p><span><span style="color:rgb(0, 0, 0)">By </span><span style="color:rgb(17, 85, 204)"><a href="https://www.researchgate.net/profile/Tommy_Boshkovski">Tommy Boshkovski</a></span></span></p>
<p style="text-align:left;display:block;"><span><span style="color:rgb(0, 0, 0)">---</span></span></p>
<p style="text-align:left;display:block;"><span><span style="color:rgb(0, 0, 0)">See Martijn van den Heuvel's keynote OHBM2018 lecture here: </span></span></p>
<p style="text-align:left;display:block;"><a href="https://www.pathlms.com/ohbm/courses/8246/sections/12540/video_presentations/115834" target="_blank">https://www.pathlms.com/ohbm/courses/8246/sections/12540/video_presentations/115834</a></p>
<p style="text-align:left;display:block;">---</p>
<p style="text-align:left;display:block;"><span><span style="color:rgb(0, 0, 0)">Martijn van den Heuvel heads the <a href="http://www.dutchconnectomelab.nl/" target="_blank">Dutch Connectome Lab</a>, part of the Complex Traits Genetics Lab at the VU University in Amsterdam. The goal of his research is to understand the association between brain complexity and brain function in health and disease. We had the pleasure to interview Martijn and find out more about his career, and also get a sneak preview of his keynote lecture at OHBM 2018 in Singapore.</span></span></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>Tommy Boshkovski (TB):</strong> <strong>Can you tell us a bit about your background and your lab?</strong></span></span></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>Martijn van den Heuvel (MvdH): </strong>I finished my undergrad and master’s in artificial intelligence, and Ph.D. studies in medical science at the University of Utrecht in the Netherlands. After finishing my Ph.D. I got a faculty position, and then my team and I recently moved to Amsterdam to the Center for Neurogenomics and Cognitive Research. There I built </span><a href="http://www.dutchconnectomelab.nl/" target="_blank"><span style="color:rgb(17, 85, 204)">my lab</span></a><span style="color:rgb(0, 0, 0)"> on connectomics, we are quite a group of connectome enthusiasts;  some of them are Ph.D. students, and some of them are postdocs now. The group is really multidisciplinary. We have a biologist, a physicist, mathematicians, psychologists, and engineers, and that's really my approach. I always try to make the group as interdisciplinary as possible, preferably with people that could do something other than what we can do.</span></span></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>TB: What motivated you to start a career in Connectomics?</strong></span></span></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>I’m not sure if anyone has intrinsic motivations to start a career in connectomics. The reason why I started to apply network science to brain imaging was actually by accident. I have a background in artificial intelligence, and I was trained in machine learning and neural networks, so to me that looked like a very obvious thing to do. I started to play around with small world network analysis, and then I got in contact with the big people in the field like Kees Stam and Olaf Sporns, and basically, that is how I rolled into the field.</span></span></p>
<p><strong><span style="color:rgb(0, 0, 0)">TB: You are one of the keynote speakers at the upcoming OHBM annual meeting. Could you tell us how you got started with OHBM?</span></strong></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>I think OHBM was my first meeting that I have ever, attended, back when I was a master’s student. I was honored that my professor let me go (I think the meeting was in Budapest that year). It was awesome to be around all those people that were working on similar things. But I always had to choose between OHBM, ISMRM and SFN, and I think later on in my career I had the luxury to sometimes go to all three, and I really like that mix. The nice thing about OHBM is that it is very application-driven, so there are many people that are using the tools, but there is also (in more recent years) a growing interest in developing new tools in the field of network science and connectomics. I am part of a joint effort with Andrew Zalesky and Alex Fornito, organizing an (almost) annual </span><a href="https://www.humanbrainmapping.org/files/2018/ED/Network%20Neuroscience.pdf"><span style="color:rgb(17, 85, 204)">educational symposium</span></a><span style="color:rgb(0, 0, 0)"> on graph theory and network science at the meeting, which is great.</span></span></p>
<p><strong><span style="color:rgb(0, 0, 0)">TB: What can we expect from your plenary lecture?</span></strong></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>I’m going to tell quite a bit about the connectome. First I’m looking forward to providing a bird’s-eye view on the connectome field. I want to spend some time on “why are we doing this?” – so that’s the first part. Ideally, I want to put this into the framework of “if the brain is wired like it is, then does this also give certain vulnerabilities to brain disorders?” One of the ways that we are looking into this is by means of comparative connectomics. We got more and more interested in looking at connectome features shared across species. If these features are evolved through evolution and are present in many different species, then they might form the very core of the brain. I think it is really important to understand these ground rules of the brain because only then can we start understanding how the changes in the ground rules may lead to various different types of brain disorders. This will make up the second part of my talk, where I won’t focus just on one disorder, but, rather, I’ll talk about the wide range of brain disorders. Most of these studies are single disorders studies (i.e. we found property X or property Y to be involved in schizophrenia) but I think the field is ready now to start looking into more deep-rooted questions. I’m asking which of these effects are specific, and which of them are common across brain disorders. It is really important to understand the multidimensional aspects and multi-disease effect of connectomics.</span></span></p>
<p><strong><span style="color:rgb(0, 0, 0)">TB: Which recent developments in the field excite you the most?</span></strong></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>There are quite a number of them, but what I really like is that the field moves a little bit away from just studying the connectome and starts combining this with other types of measurements.  The connectome field is 10 - 15 years old, but it’s pretty nice to see that people already take it for granted. I am particularly excited by using for example gene expression data and combining them with connectome data, or using cell biology data, like cytoarchitectonics, and incorporating it into the connectome model. I think these developments are pretty cool to see from a multidimensional neuroscience perspective.  What I also find exciting to see is that there is a lot OHBMof effort going on in improving the connectome field. Recent statistics papers show that it might be relatively easy to make a network, but then to study it in a very meticulous way, to have good case-control studies, it might actually be more difficult than we earlier thought. So there are great improvements in the last couple of years in new statistical methods that really also adopt the network perspective. Finally, there are so many efforts going into mapping connectomes of a wide range of species across different resolutions. A big part of the connectomics community is outside the MRI community, and I really hope that these communities will start to mix.</span></span></p>
<p><strong><span style="color:rgb(0, 0, 0)">TB: Where do you see connectomics in the next 5-10 years?</span></strong></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>Connectomics is a young field, so looking into the future 5 or 10 years, or even 5 or 10 days is difficult. I do think that there will be more room for combining different types of data into the connectome model. We’re going to see more and more papers that combine EEG with fMRI, or diffusion with functional measurements or even beyond that, such as combining genetics with imaging, because the connectome alone is not going to give us the final answer. I hope that we are going to start using in-vivo connectomics, in a more clinical application. When we start to merge the machine learning field, the big data field, with the in vivo neuroimaging connectomics, there is room for very exciting discoveries: for example, some sort of personalized connectomics where we could use connectome imaging to make predictions on disease outcome, medications response, etc.,  and I think that could be feasible. It is going to be a rough road, and I’m not sure whether we are going to do this in 5 or 10 years, but I think it’s possible to use a clinical connectome approach on an individual level, like precision connectomics.</span></span></p>
<p style="text-align:left;"><strong><span style="color:rgb(0, 0, 0)">TB: In your opinion, what are the most burning issues that connectomics is facing?</span></strong></p>
<p style="text-align:left;"><span><span style="color:rgb(0, 0, 0)"><strong>MvdH: </strong>Of course, there are issues that need to be resolved, but I’m a positive guy and I think the glass is always three quarters full. For example, there are the issues of statistics I mentioned before. We could really learn from other fields that use complexity science. That is actually where the field started, so perhaps we should go back a little bit to the roots and also look at the social networks, transportation networks, the field of finance, the field of physics and look at the methods that are developed there and get some sort of a collaboration. I think that would really help the field even further. Also, there is a lot of room for validation of the methodology that we use. The connectome field now is mostly computational, so we extract these features from the brain like hubs or modules etc., but I think it would be really cool if the field could now make the next step and start looking into hardcore biology and look for whether we see some evidence of these things there. Now it’s time to actually use cell biology research or wet-lab research to zoom into those hubs and see what is it that makes these regions so special and what are their cell properties, their genetic properties, and in that way can we validate our computational findings. I think that will really help to further ground the field into the rest of neuroscience.</span></span></p>
<p><strong><span><span style="color:rgb(0, 0, 0)">TB: What are your other interests besides connectomics?</span></span></strong></p>
<p><span><span style="color:rgb(0, 0, 0)"><strong>MvdH:</strong> I’m Dutch, so obviously I cycle just like 17 million of my fellow countryman. I’m also quite interested in tech, like new gadgets or new developments in the machine learning and/or big data fields. Besides that, I’m a dad. I have two kids, which are of course the two nicest connectomes out there.</span></span></p>
