<p><a href="../contributors.html" target="_blank">by Ilona LIpp</a></p>
<p style="display: block">Of course we all know that the brain functions as a network, but it is not straightforward to model it as such. One person who works very hard for us to be able to do so is Alex Fornito. He is a professor at Monash University and one of the leading forces in MRI-based network neuroscience. As he is also one of this year’s virtual meeting’s keynote speakers, I had the pleasure to invite Alex to a virtual meeting to ask about his scientific life.</p>
<p style="display: block"><span style="font-weight: 700">Ilona Lipp (IL): Thanks for joining me during these crazy times. Apart from OHBM going virtual, what else has changed in your scientific life in the last few weeks?</span></p>
<p><span style="font-weight: 700">Alex Fornito (AF):</span> Yes, these are unusual times. Probably the biggest change in my life has been the intimate relationship that I've developed with Zoom! But, seriously, I feel fortunate that not much has actually changed for me from a professional or scientific standpoint. A lot of the work that we do in the lab focuses on data analysis and modelling, which is reasonably straightforward to do from home. We have two young kids at home and so our regular rhythm has been disrupted because we're juggling homeschooling and work. But at the same time, it's nice to spend more time with the family, and see and help the kids learn. Relative to the disruptions that other people have had to deal with, I think I have been very lucky. I guess the main challenge is really in trying to maintain a sense of connectivity, communication, and cohesion within the lab. But I'm very fortunate to work with a fantastic team of people that make that really easy.</p>
<p><span style="font-weight: 700">IL: A main component of your research has been on connectomics, developing metrics to describe the whole brain as a network and applying them to psychiatric diseases, such as schizophrenia. How did you end up in this research niche?</span></p>
<p><span style="font-weight: 700">AF: </span>Well, that’s a bit of a long story! I did my PhD in a psychiatric lab focused on structural brain imaging where I was working on mapping cortical thickness changes in psychotic disorders. At that time, surface-based approaches were only beginning to be applied to MRI. And so this was a very exciting new way of looking at structural brain changes. But in my spare time I read a lot of fMRI work.  At the time, classic voxel-wise activation mapping was the main approach that was being used. This work was really giving us a lot of insights about how brain regions respond to different tasks, but I always felt like it provided an incomplete picture, because we know that the brain is essentially an interconnected network. I was a really big fan of those early seminal papers on connectivity by people like<span style="font-weight: 700"> </span>Olaf Sporns and<span style="font-weight: 700"> </span>Rolf Kötter, Klaas Stephan, Karl Friston, and Randy McIntosh.<span style="font-weight: 700"> </span>But the applications of network methods to imaging data were limited.</p>
<p>Then, as I was nearing the end of my PhD, I came across a paper by Sophie Achard and Ed Bullmore, which was one of the first that generated these whole brain maps of connectivity using fMRI data. I remember a figure in that paper that had a tangled graph showing how different brain regions connected to each other, and I just thought to myself, perhaps naively at the time, “that looks more like how the brain works! I want to learn how to do that!” And so I got in contact with Ed and he was gracious enough to host me for a postdoc fellowship. It was really great timing to be there, as Ed was developing his Networks Group and I was able to work with and learn from some really great people like Dannielle Bassett, David Meunier, Manfred Kitzpichler and Aaron Alexander Bloch. That experience really ended up shaping the trajectory that I ended up taking</p>
<p><span style="font-weight: 700">IL: Can you explain how looking at the brain’s connectome with MRI can help us gain a better understanding of psychiatric diseases and develop hypotheses about disease mechanisms and treatment options?</span></p>
<p><span style="font-weight: 700">AF:</span> I guess for me, it's a simple chain of logic. If we think about the brain as a network, then an important first step is to map and understand how different parts of the brain connect with each other. That's not to say that generating a map of connectivity on its own is sufficient; we also need to understand the dynamics that unfold on connectomes. But I do think that generating such a map is a necessary and important first step.</p>
<p>The first wave of connectomics studies that we've seen have been really useful for mapping where connectivity differences are between a given patient group and healthy controls. Now, in general, in psychiatry we do need to be a bit smarter about the way we define our clinical phenotypes in the first place, but we are starting to build a picture of how different brain systems are disrupted in different disorders. So now as we move into the next phase, the challenge is going to be to use this knowledge to generate new mechanistic insights and develop new treatment strategies. We're starting to see some success already, such as with the development of connectivity-guided brain stimulation protocols for mood disorders.</p>
<p>An advantage of the connectomic approach is that it can be coupled with biophysical models of brain dynamics, like neural mass models, which allow us to generate whole brain simulations of neural activity. This is an area that is still very much a work in progress, but, in principle, these models will allow us to test different mechanistic explanations for different disorders by tuning model parameters and seeing if the model can reproduce the activity changes seen in a given patient group. I think there's a lot of promise in this regard.</p>
<p><span style="font-weight: 700">IL: You have been studying schizophrenia a lot. Why is the connectome particularly interesting in this disease?</span></p>
<p><span style="font-weight: 700">AF: </span>That's a good question. So the name itself–– schizophrenia––implies a splitting or a breakdown of the mind's thought processes. And so then an obvious neurobiological hypothesis would be that this disorder emerges or arises from a disruption in the way different parts of the brain communicate with each other. This is not a new idea –– It was first suggested by Carl Wernicke over 100 years ago.</p>
<p>Personally, I think there's a natural alignment between this idea and the phenomenology of the disorder, given that it really does seem to involve a breakdown in the brain's ability to think coherently and in an organised way. We now have the tools available to really interrogate these connectivity disruptions across the entire brain. You can see how, as these approaches have developed, the thinking in the field has changed. When I was doing my PhD, most of the literature focused on the role of individual brain regions like the dorsolateral prefrontal cortex or the striatum or hippocampus; and now we see a greater emphasis on trying to understand how all these regions interact in a connected system. The hope is that these network-based understandings provide a more accurate description of what's actually happening in the disease.</p>
<p>But this doesn’t just apply to schizophrenia. We know from a lot of imaging and lesion studies that there's no single causal lesion for psychotic illness, which then leads to the idea that there is something happening at the level of interconnected circuits. This tends to be a recurring theme in a lot of psychiatric disorders. We now know that most of them can't be explained by focal damage in any single part of the brain. It is possible that at least some of these disorders might have a focal onset of pathology in one part of the brain that then spreads to affect other areas over time. Other disorders might have a truly multi-focal origin. It’s also possible that many psychiatric disorders are the result of subtle neurodevelopmental changes in brain wiring. But these are still open questions.</p>
<p><span style="font-weight: 700">IL: The microstructure and gene expression of cortical regions seems to play a large role in determining inter-cortical connections. Can you tell us a bit about the recently trending transcriptomic brain atlases and why and how you have been using them in your research?</span></p>
<p><span style="font-weight: 700">AF: </span>Well, I don't want to speak for other people, but I feel like if you spend enough time doing brain imaging, you eventually get to a point where you start to question what it is that you're actually measuring. And I mainly work with MRI, which is a fantastic tool, but it often provides indirect measures of the underlying physiological processes that we're interested in. This poses two major problems. The first is that it can be difficult to disentangle neural contributions from other contributions to the signal, including different noise sources, and that can make it difficult to interpret our findings. The second problem is that, even if we can rule out measurement noise, we often don't know what the underlying molecular mechanisms are that are driving our results. For me, the gene expression atlases provide an opportunity to try and move beyond just using the imaging data to develop some hypotheses about those underlying mechanisms.</p>
<p>That's not to say that the expression data are some kind of gold standard. In our lab, we've concentrated a lot on trying to understand some of the issues associated with gene expression data and developing workflows for how they can best be integrated with imaging measures. But I do think that if we put those issues aside and we do get a correlation between an imaging measure and the expression profile of a gene or a set of genes, then we can limit the range of possible explanations and identify candidate mechanisms that we can then pursue in further work. So it's really a way of moving beyond just mapping so that we can say: “Of all the possible molecular mechanisms that could explain what I see in this map, the expression data now allows me to narrow my search down to this set of mechanisms or pathways."</p>
<p>In our own work, we've looked at how gene expression profiles relate to brain connectivity. Other groups have done some really interesting work looking at transcriptional correlates of the effects of normal development or different types of disease. I find this work interesting because it does allow us to move past simply mapping where changes are occurring to start developing some plausible hypotheses about the specific molecules or pathways that might be involved.</p>
<p><span style="font-weight: 700">IL: Recently, the usefulness of relating variants of candidate genes to brain and behavioural phenotypes in the context of psychiatric disease has been heavily questioned. Could you tell us a bit about where this debate is coming from? What do you think are the consequences and alternatives for researchers trying to understand the genetic underpinnings of individual differences in brain structure and function?</span></p>
<p><span style="font-weight: 700">AF:</span> More or less two decades ago, the main way to identify risk variants for disease was through linkage analysis. This required people to recruit extended pedigrees and it worked well for Mendelian traits, but a lot of psychiatric disorders are not Mendelian. So researchers started to hypothesize which specific genes might be involved based on what they knew about the physiology of those disorders. And then the idea was to identify a specific variant in that gene that was known to be functional and to examine how that variant relates to some imaging or behavioural measure.</p>
<p>After a little while people started to question the plausibility of that approach, for a few reasons.  One is that the prior probability of correctly choosing a causal variant is quite low if you think about all the possible genes and variants that could contribute to complex phenotypes like schizophrenia. And we also know relatively little about the molecular mechanisms of what might be causing variation in these phenotypes. We then started to see a number of well powered studies fail to replicate earlier findings that had been published in smaller samples.</p>
<p>The solution to these and other problems in the field of genetics - because they had a false positive problem with candidate gene associations - involved shifting towards conducting large scale genome wide association studies, or GWAS, where the idea is that you compare allele frequencies between, say, a patient or control group, at hundreds of thousands or even millions of markers scattered throughout the genome. And given that you're doing so many tests and that you're often doing a Bonferroni correction over a million comparisons, you need huge sample sizes to be able to identify anything as being significant with a decent degree of power. So the sample size is generally in the order of tens of thousands of people.</p>
<p>We've had a wave of these studies now, and they've been really important in showing us that psychiatric disorders, and even brain imaging phenotypes, have a complex genetic basis. The effects of any individual variant, at least if we're talking about common variants in the population, are pretty small, at around 1%. The upshot of these developments is that if someone's interested in identifying genetic variants related to a phenotype, they probably need to conduct a GWAS as a first step. The ENIGMA consortium has really led the charge in this space with respect to imaging phenotypes, and I'm sure we'll start to see more of this kind of work as large open datasets like UK Biobank become increasingly available and used widely.</p>
<p>Personally, I view these analyses as a first step to identify which variants are related to a phenotype. But then the next step is to identify the biological effects of those variants and imaging can be helpful in addressing this goal. There are also some other really nice resources, such as data made available by the GTEx and PsychENCODE consortia, which allow people to identify which variants impact gene expression in the brain. These can be combined with data from gene expression atlases to develop a more comprehensive picture of the relationship between genes and brain. This approach aligns with the strategy we've been using in our own lab. We've tried to combine these and other sources of information to try to understand how genes influence brain connectivity.</p>
<p><span style="font-weight: 700">IL: Your research combines expertise from various disciplines, including brain imaging methodology, modelling, psychiatry, genetics etc. What are the challenges when doing so and what recommendations do you have for people who want to pursue highly interdisciplinary research?</span></p>
<p><span style="font-weight: 700">AF: </span>It's a good question. I think the interdisciplinary nature of my research probably stems from my inability to focus on a single topic! But my personal view is that the mapping between brain and behaviour is so complex, and our measurements are so imprecise, that any single approach on its own is not going to be sufficient to really tackle interesting neuroscientific questions in a comprehensive way. So the main thing that motivates and excites me about science is the opportunity to learn new ideas and get exposed to completely different ways of thinking. And so I guess I just like to explore my interests and see where they lead me.</p>
<p>I feel that the main challenge is that you always feel like a novice. Each new area or new field has its own jargon and concepts and methods and conventions, and these can take time to learn. And so I guess the best advice I could give would be to learn to be comfortable with the discomfort of not being an expert and having to start from scratch. Something that can really help with that is to team up with people who are experts in the domain and to learn as much as you can from them. Try to cultivate a good working, respectful relationship and to not be afraid to ask dumb questions, which I happen to do a lot of. Especially in the beginning of a collaboration with someone from a different discipline, you might be speaking completely different languages and it can take some patience and time to navigate those differences. But personally I find that the end result is always worth it.</p>
<p><span style="font-weight: 700">IL: </span><a href="can-you-have-it-all-a-story-about-work-life-balance-in-academia.html"><span style="color: rgb(17, 85, 204); font-weight: 700">We previously talked</span></a><span style="font-weight: 700"> about how important a healthy work-life balance is to stay productive. Leading your own research group, how do you encourage the people in your team to sometimes work less?</span></p>
<p><span style="font-weight: 700">AF:</span> I think the most important thing that someone can do is to set aside some time each day to try and do something pleasurable that is unrelated to work. That could be playing a sport or a musical instrument or doing gardening or sketch art or stamp collecting or whatever. For me, daily exercise is really important, but for others, it could be something completely different.</p>
<p>The first thing I always suggest is to create that time each day. But everyone is different, and some people struggle with that. So ultimately, I'll let people decide what's going to work for them. But sometimes, when I suggest it, people think, ‘Oh, my God, I can't do that. It's impossible. How can I spare an hour a day?’. But you never realise that you're able to do it unless you actually do it. I always think of a line from the movie The Matrix, where one of the characters says ‘You cannot ever have time if you do not make time’. And I think that's very true. Once you create that headspace, it allows you to think a bit more rationally about how you're using your time effectively. And even more broadly, where you want to go with your career, what are the things you want to focus on. Creating that healthy space can help you get a bit more perspective.</p>
<p><span style="font-weight: 700">IL: Somebody once told me that one should have a 10 year career plan ready. What are your plans for the next ten years?</span></p>
<p><span style="font-weight: 700">AF: </span>I guess it is challenging to develop a detailed 10 year plan, but I do think it is good to have long-term goals and a 10-year horizon can act as an anchor for more detailed shorter-term plans. I like to work in five-year increments.</p>
<p>If you press me, I'd have to say that there are two big questions that I want to focus on over the next 10 years. The first question is: why is the brain connected the way it is? We know that connectivity between brain regions is not random, so what are the underlying principles that govern how different parts of the brain connect to each other? Are these principles instantiated through genetic factors or other mechanisms? Do these principles or wiring rules have any bearing on our understanding for mental illness? So that's one area.</p>
<p>The other is a little more clinical and is really focused on whether we can develop an empirically-guided alternative to the DSM (Diagnostic and Statistical Manual of Psychiatric Disorders). Thinking about questions like how can we best draw the line between mental illness and health? What's the underlying latent structure of psychopathology? If we had an alternative to DSM, would it allow us to generate better insights into the biology of mental illness? These are the two big picture areas that I'm interested in, and which will frame my work over the next 10 years.</p>
<p><span style="font-weight: 700">IL: Last but not least, do you want to give us a little teaser about your OHBM virtual keynote lecture?</span></p>
<p><span style="font-weight: 700">AF:</span> I'll be talking about work we've been doing in the lab, trying to understand brain network hubs. Hubs are highly connected parts of the brain, and it's thought that they play a really important role in promoting integrated brain function.  In our lab, we've been focused on trying to understand why they get wired in the way they are, so I'll talk about work we have been doing on how to map and describe properties of brain network hubs, some of the insights that we've gained from generative models of network wiring, and what these models reveal in terms of what can and can't explain hub connectivity. I'll talk about some more recent work we've been doing focused on the genetics of hub connectivity, and I’ll present some data that suggests that there really is something quite unique and special about hubs at the level of genes. This is work that we've done across mouse, human and C. elegans, and it's trying to bring together imaging, genetics, and modelling, so hopefully there will be something in there for everyone!</p>
<p><span style="font-weight: 700">IL: Thanks a lot, I am looking forward to seeing your keynote lecture!</span></p>
